src/features/embalm/stepContent/hooks/useCreateSarcophagus/useUploadDoubleEncryptedFile.ts

```diff
@@ -20,7 +20,7 @@ export function useUploadDoubleEncryptedFile() {
       // Step 1: Encrypt the inner layer
       const encryptedInnerLayer = await encrypt(
         recipientState.publicKey,
-        Buffer.from(JSON.stringify(payload))
+        Buffer.from(JSON.stringify(payload), 'hex')
```

Comment:
Will need to do e2e tests to verify the archaeologist service can decode the data with its current codebase.